backend/api.test/EventHandlers/TestInspectionFindingsEventHandler.cs

```diff
@@ -1,26 +0,0 @@
-﻿using System;
-using System.Globalization;
-using Api.EventHandlers;
-using Xunit;
-
-namespace Api.Test.EventHandlers
-{
-    public class TestInspectionFindingsEventHandler
-    {
-        [Fact]
-        public void TestGenerateReportFromFinding()
```

Comment:
Why is the test removed?

Reply:
Because we are not using that function anymore, I can add more test for the existing functions.

Reply:
Added new test.